app/script/components/participantAvatar.js

```diff
@@ -124,26 +130,18 @@ z.components.ParticipantAvatar = class ParticipantAvatar {
 
 ko.components.register('participant-avatar', {
   template: `
-    <!-- ko if: isUser -->
-      <div class="user-avatar" data-uie-name="user-avatar" data-bind="attr: {title: participant.name}, css: cssClasses(), click: onClick, in_viewport: onInViewport, delay: delay">
-        <div class="user-avatar-background"></div>
-        <div class="user-avatar-initials" data-bind="text: initials"></div>
-        <div class="user-avatar-image participant-avatar-image"></div>
-        <div class="user-avatar-badge"></div>
-        <div class="user-avatar-border"></div>
-      </div>
-    <!-- /ko -->
-    <!-- ko if: isService -->
-      <div class="service-avatar" data-uie-name="service-avatar" data-bind="attr: {title: participant.name}, click: onClick, in_viewport: onInViewport, delay: delay">
-        <div class="service-avatar-placeholder">
-        <svg width="32" height="32" viewBox="0 0 32 32">
-          <path d="M10.5 12A6.5 6.5 0 0 0 4 18.5V24a1 1 0 0 0 1 1h22a1 1 0 0 0 1-1v-5.5a6.5 6.5 0 0 0-6.5-6.5h-11zm-7.12-1.22L.24 4.95a2 2 0 1 1 3.52-1.9L6.8 8.68C7.94 8.24 9.19 8 10.5 8h11C27.3 8 32 12.7 32 18.5V24a5 5 0 0 1-5 5H5a5 5 0 0 1-5-5v-5.5c0-3.05 1.3-5.8 3.38-7.72zM11 19a2 2 0 1 1-4 0 2 2 0 0 1 4 0m7 0a2 2 0 1 1-4 0 2 2 0 0 1 4 0m5 2a2 2 0 1 1 0-4 2 2 0 0 1 0 4zm5.26-9.55a2 2 0 0 1-3.52-1.9l3.5-6.5a2 2 0 0 1 3.52 1.9l-3.5 6.5z"/>
-        </svg>
-        </div>
-        <div class="service-avatar-image participant-avatar-image"></div>
+    <div class="participant-avatar" data-bind="attr: {title: participant.name}, css: cssClasses(), click: onClick, in_viewport: onInViewport, delay: delay">
+      <div class="avatar-initials" data-bind="text: initials"></div>
+      <div class="avatar-service-placeholder">
+          <svg width="32" height="32" viewBox="0 0 32 32">
+            <path d="M10.5 12A6.5 6.5 0 0 0 4 18.5V24a1 1 0 0 0 1 1h22a1 1 0 0 0 1-1v-5.5a6.5 6.5 0 0 0-6.5-6.5h-11zm-7.12-1.22L.24 4.95a2 2 0 1 1 3.52-1.9L6.8 8.68C7.94 8.24 9.19 8 10.5 8h11C27.3 8 32 12.7 32 18.5V24a5 5 0 0 1-5 5H5a5 5 0 0 1-5-5v-5.5c0-3.05 1.3-5.8 3.38-7.72zM11 19a2 2 0 1 1-4 0 2 2 0 0 1 4 0m7 0a2 2 0 1 1-4 0 2 2 0 0 1 4 0m5 2a2 2 0 1 1 0-4 2 2 0 0 1 0 4zm5.26-9.55a2 2 0 0 1-3.52-1.9l3.5-6.5a2 2 0 0 1 3.52 1.9l-3.5 6.5z"/>
+          </svg>
```

Comment:
I'd prefer toggling these with knockout instead of bloating the dome with display none elements